.github/workflows/build-and-publish.yml

```
@@ -0,0 +1,167 @@
+    branches: [ main, develop ]
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - name: Checkout code
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)